hedera-mirror-rest/__tests__/integration.test.js

```diff
@@ -229,7 +229,7 @@ describe('API specification tests', () => {
     const specMap = new Map();
 
     walk(specPath)
-      .filter((f) => f.endsWith('.json'))
+      .filter((f) => f.endsWith('account-id.json'))
```

Comment:
revert temporary change

```suggestion
      .filter((f) => f.endsWith('account-id.json'))
```

Reply:
Thanks for spotting this, I have reverted it.